content/news/041/index.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "This Month in Rust GameDev #41 - December 2022"
+transparent = true
+date = 2023-01-04
+draft = true
++++
+
+<!-- no toc -->
+
+<!-- Check the post with markdownlint-->
+
+Welcome to the 41st issue of the Rust GameDev Workgroup's
+monthly newsletter.
+[Rust] is a systems language pursuing the trifecta:
+safety, concurrency, and speed.
+These goals are well-aligned with game development.
+We hope to build an inviting ecosystem for anyone wishing
+to use Rust in their development process!
+Want to get involved? [Join the Rust GameDev working group!][join]
+
+You can follow the newsletter creation process
+by watching [the coordination issues][coordination].
+Want something mentioned in the next newsletter?
+[Send us a pull request][pr].
+Feel free to send PRs about your own projects!
+
+[Rust]: https://rust-lang.org
+[join]: https://github.com/rust-gamedev/wg#join-the-fun
+[pr]: https://github.com/rust-gamedev/rust-gamedev.github.io
+[coordination]: https://github.com/rust-gamedev/rust-gamedev.github.io/issues?q=label%3Acoordination
+
+- [Announcements](#announcements)
+- [Game Updates](#game-updates)
+- [Learning Material Updates](#learning-material-updates)
```

Comment:
I noticed during the last couple of newsletters that 'Learning Material Updates' and 'Engine Updates' are flipped in the table of contents - could we fix this (and the template) so we don't have to remember to do it later? 🙁